src/main/resources/assets/admin/common/js/ui/time/Picker.ts

```diff
@@ -12,16 +12,24 @@ import {SelectedDateChangedEvent} from './SelectedDateChangedEvent';
 export class Picker<T extends Element>
     extends DivEl {
 
+    private showDefaultButton: Boolean = false;
+
     protected popup: T;
 
     protected popupOkButton: Button;
 
+    protected popupDefaultButton: Button;
```

Comment:
We don't need this as a class property, just like we don't need `popupOkButton` as a class property. `init` methods return an instance of each button which is then used inside `createPopup`.